docs/reference/data-frames/apis/stop-transform.asciidoc

```diff
@@ -0,0 +1,45 @@
+[role="xpack"]
+[testenv="basic"]
+[[stop-data-frame-transform]]
+=== Stop {dataframe-transforms} API
+++++
+<titleabbrev>Stop {dataframe-transforms}</titleabbrev>
+++++
+
+Stops one or more {dataframe-transforms}.
+
+==== Request
+
+`POST _data_frame/transforms/<data_frame_transform_id>/_stop`
+
+//==== Description
+
+==== Path Parameters
+
+`data_frame_transform_id` (required)::
+  (string) Identifier for the {dataframe-transform}. This identifier can contain
+  lowercase alphanumeric characters (a-z and 0-9), hyphens, and underscores. It
+  must start and end with alphanumeric characters.
+
```

Comment:
can you add query parameters `wait_for_completion` and `timeout`? Both work the very same way as documented for stop rollup,